apps/meteor/ee/client/omnichannel/slaPolicies/SlasRoute.tsx

```diff
@@ -29,39 +36,41 @@ const useQuery = ({ text, itemsPerPage, current }, [column, direction]) =>
 		[text, itemsPerPage, current, column, direction],
 	);
 
-function PrioritiesRoute() {
+function SlasRoute(): ReactElement {
 	const t = useTranslation();
-	const canViewPriorities = usePermission('manage-livechat-sla');
+	const canViewSlas = usePermission('manage-livechat-sla');
 
-	const [params, setParams] = useState({ text: '', current: 0, itemsPerPage: 25 });
-	const [sort, setSort] = useState(['name', 'asc']);
+	const [params, setParams] = useState<GenericTableParams>({ text: '', current: 0, itemsPerPage: 25 });
+	const [sort, setSort] = useState<[string, 'asc' | 'desc']>(['name', 'asc']);
 
 	const debouncedParams = useDebouncedValue(params, 500);
 	const debouncedSort = useDebouncedValue(sort, 500);
-	const query = useQuery(debouncedParams, debouncedSort);
-	const prioritiesRoute = useRoute('omnichannel-priorities');
+	const query = useEndpointQuery(debouncedParams, debouncedSort);
+	const SlasRoute = useRoute('omnichannel-sla-policies');
 	const context = useRouteParameter('context');
-	const id = useRouteParameter('id');
+	const id = useRouteParameter('id') || '';
 
-	const onHeaderClick = useMutableCallback((id) => {
+	const onHeaderClick = useMutableCallback((id: string) => {
```

Comment:
Why this was not removed, if is not used?

Reply:
@tiagoevanp I must've missed when removing the unused code.

Reply:
@tiagoevanp I double checked, It is being used. If you are using Github as base, remember that unchanged code remains hidden.

Reply:
Yes, I think something just escaped from my eyes... 😅